src/lib/shopping-fpti/event-handlers.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+import type { Config } from '../../types';
+import type { EventType } from '../../types/shopping-events';
+
+export function eventEnricherInit(config : Config) : Object {
+  function enrichPageViewEvent() : Object {
+    const offerId = config.containerSummary ? config.containerSummary.programId : null;
+    if (offerId) {
+      return {
+        es: 'visitorInfoFlowStarted',
+        fltp: 'store-cash',
+        offer_id: offerId
+      };
+    }
+    return {};
+  }
+
+  function enrichPurchaseEvent() : Object {
```

Comment:
Why does it named **enrich**PurchaseEvent ?

Reply:
it return new FPTI attributes based on the event type